src/DbBackupsTrackerController.cpp

```diff
@@ -160,18 +160,13 @@ void DbBackupsTrackerController::askForExportBackup()
 
 void DbBackupsTrackerController::exportDbBackup()
 {
-    QString format;
```

Comment:
are you discarding the file format set by the user when the user is clicking the export button? 
Because this is is not what I asked.... this code is correct, you may simply correct the typo

Reply:
not exactly. class DbBackupsTrackerController initiates a backup prompt when DB in a device is more recent than the file.
User choice is safe: it in https://github.com/mooltipass/moolticute/blob/master/src/MainWindow.cpp#L1164